x-pack/plugins/uptime/public/components/functional/ping_list.tsx

```diff
@@ -49,46 +50,38 @@ export const PingList = ({
       name: i18n.translate('xpack.uptime.pingList.statusColumnLabel', {
         defaultMessage: 'Status',
       }),
-      render: (pingStatus: string) => (
-        <EuiHealth color={pingStatus === 'up' ? 'success' : 'danger'}>
-          {pingStatus === 'up'
-            ? i18n.translate('xpack.uptime.pingList.statusColumnHealthUpLabel', {
-                defaultMessage: 'Up',
-              })
-            : i18n.translate('xpack.uptime.pingList.statusColumnHealthDownLabel', {
-                defaultMessage: 'Down',
-              })}
-        </EuiHealth>
+      render: (pingStatus: string, item: Ping) => (
+        <div>
+          <EuiHealth color={pingStatus === 'up' ? 'success' : 'danger'}>
+            {pingStatus === 'up'
+              ? i18n.translate('xpack.uptime.pingList.statusColumnHealthUpLabel', {
+                  defaultMessage: 'Up',
+                })
+              : i18n.translate('xpack.uptime.pingList.statusColumnHealthDownLabel', {
+                  defaultMessage: 'Down',
+                })}
+          </EuiHealth>
+          <EuiText size="xs" color="subdued">
+            Latest was {moment(item.timestamp).fromNow()}
+          </EuiText>
+        </div>
       ),
     },
-    {
-      field: 'timestamp',
-      name: i18n.translate('xpack.uptime.pingList.timestampColumnLabel', {
-        defaultMessage: 'Timestamp',
-      }),
-      render: (timestamp: string) => moment(timestamp).fromNow(),
-    },
     {
       field: 'monitor.ip',
+      dataType: 'number',
       name: i18n.translate('xpack.uptime.pingList.ipAddressColumnLabel', {
         defaultMessage: 'IP',
       }),
     },
-    {
-      field: 'monitor.id',
-      name: i18n.translate('xpack.uptime.pingList.idColumnLabel', {
-        defaultMessage: 'Id',
-      }),
-      dataType: 'string',
-      width: '20%',
-    },
     {
       field: 'monitor.duration.us',
       name: i18n.translate('xpack.uptime.pingList.durationMsColumnLabel', {
-        defaultMessage: 'Duration ms',
+        defaultMessage: 'Duration',
         description: 'The "ms" in the default message is an abbreviation for milliseconds',
       }),
-      render: (duration: number) => duration / 1000,
+      dataType: 'number',
+      render: (duration: number) => <span>{duration / 1000} ms</span>,
```

Comment:
```suggestion
      render: (duration: number) => (
        <span>
          <FormattedMessage
            id="xpack.uptime.pingList.durationMsColumnLabel"
            defaultMessage="{duration} ms"
            values={{ duration: duration / 1000 }}
          />
        </span>
      ),
```

We can't specify an ms value in a bare string like this, needs to be localized.